3-mcstas/mcstas-powder-diffraction.ipynb

```diff
@@ -102,132 +103,623 @@
+   "cell_type": "markdown",
+   "id": "6dd52c67-0fe5-4ed1-a924-993a0c8e27c5",
+   "metadata": {},
+   "source": [
+    "### Set sample size\n",
+    "Select appropriate sample size given Vanadium has a macroscopic scattering cross section $\\Sigma$ of around 0.35 cm^-1. A neutron beam with intensity $I_0$ that travel in a media a distance of $z$ will be attenuated as the neutrons scatter in the material, and the remaining intensity $I$ can be calculated with the Beer-Lambert law:\n",
+    "$$\n",
+    "I = I_0 e^{-z\\Sigma}\n",
+    "$$\n",
+    "\n",
+    "For our experiment we want to observe neutrons that scattered once, as neutrons that scatter more than once would be considered background.\n",
+    "\n",
+    "What sample depth would be appropriate?\n",
+    "\n",
+    "- A: 10 cm\n",
+    "- B: 1 cm\n",
+    "- C: 1 mm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "06fdd143-1dab-4c75-a98e-1fe33bf1eabb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# answer\n",
+    "import numpy as np\n",
+    "fraction_left = np.exp(-0.35*1)\n",
+    "print(\"scattering probability\", 1 - fraction_left)\n",
+    "print(\"single scattering probability\", (1 - fraction_left)*fraction_left)\n",
+    "print(\"double scattering probability\", (1 - fraction_left)**2*fraction_left)\n",
+    "print(\"single to multiple scattering probability \", (1 - fraction_left)*fraction_left/(1-fraction_left))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "adfea97a-ce20-457c-8c8f-39415c9696df",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "quiz.question_1(\"B\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d40d30a4-2cdd-4e5a-b584-ed6f4a9202de",
+   "metadata": {},
+   "source": [
+    "### Set sample size in simulated instrument\n",
+    "Use the *set_parameters* method to set the sample thickness, here using the parameter *sample_radius*. Set the wavelength range using \"l_min\" and \"l_max\" to 2.5 - 2.501."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "49c34751-fc94-470b-833f-bab3995c21bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wavelength = 2.5\n",
+    "instrument.set_parameters(sample_radius=0.005, l_min=wavelength, l_max=wavelength + 0.001)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a9e3cea0-137f-4bf4-8580-d9bd908eba42",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "quiz.question_2(instrument)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae1dff88-3b87-4286-a13b-0f6d4e746928",
+   "metadata": {},
+   "source": [
+    "### Run simulation\n",
+    "Run the simulation in this configuration and plot the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bfe9f1ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "instrument.settings(ncount=4.0e6, mpi=10, suppress_output=True, NeXus=True)"
```

Comment:
```suggestion
    "instrument.settings(ncount=4.0e6, mpi=4, suppress_output=True, NeXus=True)"
```
10 will most likely fail on visa instances.
If students think 4 is too slow, they can experiment with upping the number, but at least we give them a safe number to start with